### ISSUE TITLE
No Snapshot version of runtime anymore?

### ISSUE THREAD

**Issue by kutoman** [UnderwaterApps/overlap2d#257]
getting the following error:

```
   > Could not find com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx:0.1.1-SNAPSHOT.
     Searched in the following locations:
.....
         https://repo1.maven.org/maven2/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/maven-metadata.xml
         https://repo1.maven.org/maven2/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.pom
         https://repo1.maven.org/maven2/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.jar
         https://oss.sonatype.org/content/repositories/snapshots/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/maven-metadata.xml
         https://oss.sonatype.org/content/repositories/snapshots/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.pom
         https://oss.sonatype.org/content/repositories/snapshots/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.jar
         https://oss.sonatype.org/content/repositories/releases/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/maven-metadata.xml
         https://oss.sonatype.org/content/repositories/releases/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.pom
         https://oss.sonatype.org/content/repositories/releases/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.1-SNAPSHOT/overlap2d-runtime-libgdx-0.1.1-SNAPSHOT.jar
....

```

**Comment by htynkn** [comment]
overlap2d-runtime-libgdx had released 0.1.1, so there is no snapshot for 0.1.1 anymore.

The next version is 0.1.2, so we have [0.1.2-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/com/underwaterapps/overlap2druntime/overlap2d-runtime-libgdx/0.1.2-SNAPSHOT/).

**Comment by kutoman** [comment]
oh that release passed me.. thanks!
